pkgs/development/compilers/llvm/11/llvm-genode.patch

```diff
@@ -0,0 +1,58 @@
+diff --git a/include/llvm/ADT/Triple.h b/include/llvm/ADT/Triple.h
+index 6bad18f19244..95459ca997c5 100644
+--- a/include/llvm/ADT/Triple.h
++++ b/include/llvm/ADT/Triple.h
+@@ -166,6 +166,7 @@ public:
+     DragonFly,
+     FreeBSD,
+     Fuchsia,
++    Genode,
```

Comment:
Does inserting values in the middle of an enumeration break compatibility with anything we currently could interchange with LLVM on other distributions (I guess some IR kinds are interchangeable)?

Reply:
That I don't know, but generally we are using an older version of LLVM. Darwin, FreeBSD, and Linux are on 7, WASM on 8. 11 is used by Chromium, Emscripten, and Zig.